<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
      <item>
       <widget class="QTableWidget" name="tableWidget_users">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <column>
         <property name="text">
          <string>UserName</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>IP Address</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Port</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QComboBox" name="comboBox_Type_1">
            <item>
             <property name="text">
              <string>Strength</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Defense</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Shield</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Agility</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_Rarity_1">
            <item>
             <property name="text">
              <string>Common</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Rare</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Epic</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Legendary</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Create_1">
            <property name="text">
             <string>Create</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelPM_1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_Attack_1">
            <property name="text">
             <string>Attack</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_LvUp_1">
            <property name="text">
             <string>LvUp</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_LvMax_1">
            <property name="text">
             <string>LvMax</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_moveInfo_1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget_monsters">
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Level</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Attack</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Defence</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>MaxHealth</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Speed</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Exp</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Limitbreak</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Rarity</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
